content/addons.md

```diff
@@ -1,14 +1,142 @@
 # Addons
 
-<!-- What is an addon, link to Ember Observer -->
+The modern web developer has an incredible selection of open source code that they can use through the npm ecosystem, plus great tools for writing their own modules. Ember developers are free to use regular npm packages in their apps, but there are also thousands of packages that are made specifically for the Ember ecosystem. We call such packages "addons."
```

Comment:
I think it would be helpful for newcomers to understand the difference between ember add-on vs. a regular node.js library:

+ adding deps to `package.json`
+ generators
+ preprocessors
+ fingerprinting
+ etc